src/main/java/core/basesyntax/Randomizer.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Randomizer {
+    public int getRandomNumber() {
+        return (int) (1 + (Math.random() * ((5 - 1) + 1)));
```

Comment:
```suggestion
        return (int) (1 + (Math.random() * 5));
```

and let's use 5 as constant. Or you can pass it as parameter to define max number, that your `getRandomNumber` can return